js/tonnetz.js

```diff
@@ -446,6 +451,34 @@ var tonnetz = (function() {
     toneGrid[tone].push(node);
   };
 
+  var drawUnitCell = function(ctx) {
+    var A = 9; // This is tone A
+
+    // to identify the closest A note to the center of the canvas
+    var distances = $.map(toneGrid[A], function(d){
+      return Math.sqrt((d.x - W/2)**2 + (d.y - H/2)**2);
+    });
+    // index of minimum distance
+    var i = distances.indexOf(Math.min.apply(Math, distances));
+    setTranslate(ctx, toneGrid[A][i].x, toneGrid[A][i].y);
```

Comment:
Seems like a lot of computation just to determine a constant value. :slightly_smiling_face:  You can get it with `getNeighborXYDiff(0, 9)`.

Reply:
Thanks, I will correct accordingly.